<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewDriver</class>
 <widget class="QDialog" name="DialogNewDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New driver</string>
  </property>
  <widget class="QPushButton" name="btnMoreInfo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show values</string>
   </property>
   <property name="text">
    <string>More info ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUseDefaults">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set config to default
values and update the database</string>
   </property>
   <property name="text">
    <string>Use default values</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>This driver board has been added to the system.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cancel driver
configuration</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>The driver configuration has to be initialized.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUseDriver">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Update database
with current driver config</string>
   </property>
   <property name="text">
    <string>Use values in the
driver board</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExpertInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>The driver board seems to be already configured by an expert.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
